### PR TITLE
feat: Add aks ingress autoscaling + pin nginx ingress version as security best practice

### PR DESCRIPTION
**Pull request by pasqualespica**
The purpose of this PR is to : 
 - add autoscaling to nginx ingress
 - pin nginx ingress version as security best practice

### List of changes

Update helm nginx controller

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

**Review by pasqualedevita — DISMISSED** [review]
PR it's ok but also it's better to pin ngnix versions as in https://github.com/pagopa/io-infra/pull/216

File: src/aks/env/weu-dev/terraform.tfvars [hunk 1]
```diff
@@ -60,7 +60,8 @@ aks_user_node_pool = {
 
 aks_cidr_subnet = ["10.1.0.0/17"]
 
-ingress_replica_count    = "2"
-ingress_load_balancer_ip = "10.1.100.250"
-nginx_helm_version       = "4.1.0"
-keda_helm_version        = "2.6.2"
+ingress_min_replica_count = "2"
```
Comment (pasqualedevita) [inline src/aks/env/weu-dev/terraform.tfvars:63]:
```suggestion
ingress_min_replica_count = "1"
```

File: src/aks/env/weu-uat/terraform.tfvars [hunk 2]
```diff
@@ -61,7 +61,8 @@ aks_user_node_pool = {
 
 aks_cidr_subnet = ["10.1.0.0/17"]
 
-ingress_replica_count    = "2"
-ingress_load_balancer_ip = "10.1.100.250"
-nginx_helm_version       = "4.1.0"
-keda_helm_version        = "2.6.2"
+ingress_min_replica_count = "2"
+ingress_max_replica_count = "3"
```
Comment (pasqualedevita) [inline src/aks/env/weu-uat/terraform.tfvars:65]:
```suggestion
ingress_max_replica_count = "30"
```
ready for performance test

Comment (pasqualespica) [reply src/aks/env/weu-uat/terraform.tfvars:65]: right

**Comment by pasqualespica** [comment]
> PR it's ok but also it's better to pin ngnix versions as in [pagopa/io-infra#216](https://github.com/pagopa/io-infra/pull/216)

tnx, I fix it in this PR

**Pull request merged**